resources/rpm/project.spec.erb

```diff
@@ -48,6 +48,15 @@ Autoreq: 0
 <%- get_requires.each do |requires| -%>
 Requires:  <%= requires %>
 <%- end -%>
+# All rpm packages built by vanagon have the pre-/post-install script
+# boilerplates (defined below). These require `mkdir` and `touch` but previously
+# did not specify a dependency on these.
+# In the future, we will supress pre/post scripts completely if there's nothing
+# specified by the project or the components.
+Requires(pre): /bin/mkdir
+Requires(pre): /bin/touch
+Requires(post): /bin/mkdir
+Requires(post): /bin/touch
```

Comment:
It looks like this isn't isolated to el5. Will mkdir and touch be in the same path for all the platforms we make this package available on?